test/WebJobs.Script.Tests.E2E.Shared/ArmAuthenticationHelpers.cs

```diff
@@ -7,7 +7,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace WebJobs.Script.EndToEndTests
+namespace WebJobs.Script.Tests.E2E.Shared
```

Comment:
nit: for consistency with the rest of the product, use `EndToEnd` instead of `E2E`